### PR TITLE
docs(README): add aircip.ir and platform.nazarbazaar.ir to `built with` section

### PR DESCRIPTION
**Pull request by ali-master**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ These great products are built on Semantic UI React. Add yours [here][22].
 - https://thefaithcircle.com
 - https://appfollow.io
 - http://according.to.localsourc.es
+- http://aircip.ir
```
Comment (layershifter) [inline README.md]: This website returns 404 error

**Review by layershifter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ali-master) [reply README.md]: fixed it

File: README.md [hunk 2]
```diff
@@ -84,6 +84,8 @@ These great products are built on Semantic UI React. Add yours [here][22].
 - https://thefaithcircle.com
 - https://appfollow.io
 - http://according.to.localsourc.es
+- http://www.aircip.ir
```
Comment (levithomason) [inline README.md:87]: This redirects to http://www.aircip.ir/auth/login, which is a 404.

Comment (ali-master) [reply README.md:87]: Sorry, because this project runs on IIS and IIS doesn't serve react-router URLs, it returns this error.

**Comment by layershifter** [comment]
As I see both websites are working now.

**Pull request merged**